war/src/main/java/org/exoplatform/addons/es/EmbeddedESStartupServlet.java

```diff
@@ -42,27 +46,37 @@ public void init() throws ServletException {
     }
 
     LOG.info("Initializing elasticsearch Node '" + getServletName() + "'");
-    Settings.Builder settings = Settings.settingsBuilder();
+    Settings.Builder settings = Settings.builder();
 
     InputStream resourceAsStream = getServletContext().getResourceAsStream("/WEB-INF/elasticsearch.yml");
-    if (resourceAsStream != null) {
+    if (resourceAsStream == null) {
+      throw new ServletException("Error while initializing elasticsearch node, configuration file '/WEB-INF/elasticsearch.yml' couldn't be found");
+    }
+    try {
       settings.loadFromStream("/WEB-INF/elasticsearch.yml", resourceAsStream);
-      try {
-        resourceAsStream.close();
-      } catch (IOException e) {
-        // ignore
-      }
+      resourceAsStream.close();
+    } catch (IOException e) {
+      throw new ServletException("Error while initializing elasticsearch node '" + getServletName() + "'", e);
     }
 
-    if (settings.get("http.enabled") == null) {
-      settings.put("http.enabled", false);
+    String pathData = settings.get("path.data");
+    if (pathData == null) {
+      settings.put("path.data", System.getProperty("exo.data.dir"));
+    } else if (pathData.startsWith("${")) {
```

Comment:
Does it mean we will still be able to override any default ES configuration by settings them in exo.properties with the prefix "es." (as defined in the documentation : https://www.exoplatform.com/docs/public/topic/PLF44/PLFAdminGuide.Configuration.ElasticSearch.html?resultof=%22%65%6c%61%73%74%69%63%73%65%61%72%63%68%22%20) ?
If so, why changing the value of "cluster.name" for example (https://github.com/exo-addons/exo-es-embedded/pull/9/files#diff-d4893754c9fae24a1defee0099606c50R6) ?